microshift_getting_started/microshift-architecture.adoc

```diff
@@ -10,7 +10,24 @@ Learn the specifics of {product-title} architecture including design intent, how
 
 [id="microshift-architectural-design_{context}"]
 == Architectural design
-{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. {product-title} contains only the APIs and features essential to operating in edge and small form factor computing environments. The following operational differences from {OCP} can help you understand where {product-title} can be deployed:
+{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. {product-title} contains only the APIs and features essential to operating in edge and small form factor computing environments. {product-title} is tasked with only the following Kubernetes cluster services:
+
+* Networking
+* Ingress
+* Storage
+* Helm
+
+{product-title} also provides the following Kubernetes functions:
+
+* Orchestration
+* Security
+
+{product-title} is optimized when used with a compatible operating system, such as {op-system-ostree-first}. When used together, {product-title} and {op-system-ostree-first} form {op-system-bundle}. {product-title} can also be deployed as a component by users of {OCP}.
```

Comment:
The last sentence makes it sound like you would deploy MicroShift on top of OpenShift, but that's not the case. I suggest removing it. If we need to document here that OpenShift users are entitled to use MicroShift, we should find another way to say that.

```suggestion
{product-title} is optimized when used with a compatible operating system, such as {op-system-ostree-first}. When used together, {product-title} and {op-system-ostree-first} form {op-system-bundle}.
```

Reply:
Maybe something like, 
Users of {OCP} can create {product-title} deployments separately from other platform services.

Reply:
What's the goal of including any statement about OCP users at all? Are we trying to talk about how MicroShift is tied to subscriptions? Or are we trying to talk about the technical relationship between OCP and MicroShift?

Reply:
The purpose to let OCP users know they can try this out since there is no mention in the OCP docs.

Reply:
OK, I don't necessarily think this document is the best place for that. We will have blog posts, press releases, etc. for announcements like that and they will explain which subscriptions are entitled to use MicroShift.